apps/src/aichat/views/aichatView.module.scss

```diff
@@ -19,15 +19,18 @@
   .instructionsArea {
     width: 30%;
     height: 100%;
+    min-width: 200px;
```

Comment:
Mind adding a comment similar to your PR description around why this value was chosen? Optional - also, create a sass constant for this value?